.gitignore

```diff
@@ -54,3 +54,4 @@ build-iPhoneSimulator/
 
 # Used by RuboCop. Remote config files pulled in from inherit_from directive.
 # .rubocop-https?--*
+/Gemfile.lock
```

Comment:
[nit] Looks like this entry is in line 48 above ☝️ , you can just uncomment that one out?